lib/ddtrace/contrib/rails/core_extensions.rb

```diff
@@ -30,9 +30,9 @@ module CacheStoreFetchExtension
     # It might seem redundant to instrument both read and fetch since
     # fetch very often calls read. But there's no garantee of this, in
     # some cases fetch can call directly read_entry without calling read.
-    def fetch(*args)
+    def fetch(*args, &block)
```

Comment:
For what I know the ``*`` operator should unpack also the block reference. This is an example: https://gist.github.com/palazzem/49c8a0246bc6ca5a6c2776332ada0335

Am I missing something?